checker/src/org/checkerframework/checker/nullness/KeyForAnalysis.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.checker.nullness;
+
+import java.util.List;
+import java.util.Set;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.VariableElement;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.flow.CFAbstractAnalysis;
+import org.checkerframework.framework.flow.CFAbstractValue;
+import org.checkerframework.javacutil.Pair;
+
+public class KeyForAnalysis extends CFAbstractAnalysis<KeyForValue, KeyForStore, KeyForTransfer> {
```

Comment:
A quick note that everything in this class is just to glue together the other classes would safe time in the future. I looked through everything and didn't find any actual logic (which is something we should try to automate...). If I missed something, even more reason to document here.